assets/xml/objects/object_zf.xml

```diff
@@ -5,7 +5,7 @@
         <Texture Name="object_zf_Tex_000240" OutName="tex_00000240" Format="ci8" Width="16" Height="16" Offset="0x240"/>
```

Comment:
TlutOffset="0x0" got lost on this texture node and the previous one in the merge